data/src/main/java/com/globant/data/mapper/CharacterMapper.kt

```diff
@@ -0,0 +1,37 @@
+package com.globant.data.mapper
+
+import com.globant.data.service.response.CharacterResponse
+import com.globant.data.service.response.CharactersBaseResponse
+import com.globant.data.service.response.MarvelBaseResponse
+import com.globant.data.service.response.ThumbnailResponse
+import com.globant.domain.entity.Character
+
+class CharacterMapper {
+
+    private fun transformToThumbnail(thumbnailResponse: ThumbnailResponse): String =
+        "${thumbnailResponse.path}$DOT${thumbnailResponse.extension}"
+
+    private fun transformToCharacter(characterResponse: CharacterResponse): Character =
+        Character(
+            characterResponse.id,
+            characterResponse.name,
+            getDescription(characterResponse.description),
+            transformToThumbnail(characterResponse.thumbnail)
+        )
+
+
+    private fun getDescription(description: String): String {
+        return if (description.isEmpty())
```

Comment:
please add brackets ({ }) to indent the code in the if and else statement

Reply:
Opa, yes use please kotlin way for if-result

Reply:
Done, thanks guys